packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -240,7 +269,6 @@ contract GoldToken is
    * @return True if the transaction succeeds.
    */
   function _transfer(address to, uint256 value) internal returns (bool) {
-    require(to != address(0), "transfer attempted to reserved address 0x0");
```

Comment:
Is the removal of this check necessary, since to burn address is different anyways?